src/commands/Utilities/support.js

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const Discord = require('discord.js');
```

Comment:
You need to import the Discord.js package to use the embed below.
(Correctly instead of importing it for Discord)*